src/server/rdb_load.cc

```diff
@@ -2569,7 +2540,13 @@ void RdbLoader::LoadItemsBuffer(DbIndex db_ind, const ItemsBuf& ib) {
                         item->val.rdb_type);
   };
 
+  bool dry_run = absl::GetFlag(FLAGS_rdb_load_dry_run);
```

Comment:
Wouldn't that acquire a lock internally each time ? Why not simply cache this flag ? Furthermore, once we start df with this set to true, rdb loader won't load anything. Would it worth to make this flag mutable ?

Reply:
we use it only for development purposes and its affect is negligible (we grab it once for each batch)